spec/models/preservation_policy_spec.rb

```diff
@@ -54,63 +52,40 @@
         archive_policy: Config::Options.new(archive_ttl: 666, fixity_ttl: 666)
       )
       allow(Settings.preservation_policies).to receive(:policy_definitions).and_return(archive_pres_policy_setting)
-      PreservationPolicy.seed_from_config
-      expect(PreservationPolicy.find_by(preservation_policy_name: 'archive_policy')).to be_a_kind_of PreservationPolicy
+      described_class.seed_from_config
+      expect(described_class.find_by(preservation_policy_name: 'archive_policy')).to be_a_kind_of described_class
     end
   end
 
   describe '.default_policy' do
+    # clear the cache before each test (and after all) to reset
+    before { described_class.default_policy = nil }
+    after(:all) { described_class.default_policy = nil }
+
     it 'returns the default preservation policy object' do
       # db already seeded
-      expect(PreservationPolicy.default_policy).to be_a_kind_of PreservationPolicy
+      expect(described_class.default_policy).to be_a_kind_of described_class
     end
 
     it "raises RecordNotFound if the default policy doesn't exist in the db" do
-      # a bit contrived, but just want to test that lack of default PreservationPolicy causes lookup to
+      # a bit contrived, but just want to test that lack of default described_class causes lookup to
```

Comment:
this comment is the one place where keeping PreservationPolicy as the string might read better

Reply:
yup